style.css

```diff
@@ -0,0 +1,775 @@
+/* Core Styles ---------> */
+*,
+**::before,
+**::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: 'Fredoka', sans-serif;
+  font-size: 2vw;
+}
+
+@import url('https://fonts.googleapis.com/css2?family=Fredoka:wght@400;500;600;700&display=swap');
+
+:root {
+  --color-3: #ec5242;
+  --color-4: #e83611;
+  --color-5: #f9002f;
+}
```

Comment:
Thank you for your feedback @Gambit142 . These colors are for title, I did not use them anywhere else in the webpage. If I remove them it changes the design of my title and I want to see that in this way. Still do I need to remove them ?

Reply:
I understand @Fsher07 , but one of the project requirements state that we should use the colour provided by Cindy Shin on Behance. For approval purposes, I think it would be nice for you to stick to the colours required, after approval, you can change the colours and make further changes as you wish.